################################################################################
SHORT DESCRIPTION: 
################################################################################
Fix up rules when forcing -mno-sse

################################################################################
LONG DESCRIPTION: 
################################################################################
Xen-firmware had issues due to the fact that the default sysroot is poisoned.
Pulled a change from the upstream recipes in do_configure. Also pulled a Xen
patch to resolve a compile error that the SSE instruction set was disabled.
  - Fixed items that need to be package specific
  - Leverage python in sysroot
  - Cleaned up some oerunmake calls

################################################################################
CHANGELOG 
################################################################################
Author:
    Signed-off-by: Adam Oliver <aikidokatech@gmail.com>

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
--- a/xen/arch/x86/arch.mk
+++ b/xen/arch/x86/arch.mk
@@ -37,7 +37,7 @@ CFLAGS += -mno-red-zone -fpic -fno-async
 
 # Xen doesn't use SSE interally.  If the compiler supports it, also skip the
 # SSE setup for variadic function calls.
-CFLAGS += -mno-sse $(call cc-option,$(CC),-mskip-rax-setup)
+CFLAGS += -mno-sse -mfpmath=387 $(call cc-option,$(CC),-mskip-rax-setup)
 
 # Compile with thunk-extern, indirect-branch-register if avaiable.
 CFLAGS-$(CONFIG_INDIRECT_THUNK) += -mindirect-branch=thunk-extern
